resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -43,8 +43,8 @@ import org.apache.spark.internal.config._
 class KubernetesSuite extends SparkFunSuite
   with BeforeAndAfterAll with BeforeAndAfter with BasicTestsSuite with SecretsTestsSuite
   with PythonTestsSuite with ClientModeTestsSuite with PodTemplateSuite with PVTestsSuite
-  with DepsTestsSuite with DecommissionSuite with RTestsSuite with Logging with Eventually
-  with Matchers {
+  with DepsTestsSuite with DecommissionSuite with RTestsSuite with SparkConfPropagateSuite
```

Comment:
Shall we put this after `BasicTestsSuite` or `SecretsTestsSuite`?

Reply:
Yes!